technologies/app/rshiny/rshiny-1.7.1/ressources/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+echo "SAAGIE_BASE_PATH = $SAAGIE_BASE_PATH"
```

Comment:
Maybe we can add a message explaining how to use R Shiny (which environment variables to set) so taht users will understand why their app is failing.

Reply:
Just a message like in the Readme or you mean also to test that the mandatory variables are set (eg: SHINY_GIT_BRANCH) ?

Reply:
I'll do both

Reply:
I prefer to have both because when you're launching an App from Saagie, you don't really know that there is a README with specific instructions.